config/build.config.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 var versionFile = __dirname + '/../dist/commit';
 
 module.exports = {
-  ngVersion: '1.3.15',
+  ngVersion: '1.4.3',
```

Comment:
Yep, I think that's good - as long as it's stable, we should be running against the latest version of Angular.